src/services/virtual-machines-service.js

```diff
@@ -19,6 +19,16 @@ class VirtualMachinesService {
     return vprotectApiService.get('/virtual-machines/' + id);
   }
 
+  getVirtualMachinesPage(filter, page, size) {
+    vprotectApiService
+      .get(
+        `/virtual-machines?filter=${filter}&page=${page}&size=${size}`,
```

Comment:
We shouldn't pass hard coded params 
getVirtualMachinePage(params){
get('/url', {params})
}